state/protocol/badger/state.go

```diff
@@ -849,24 +849,11 @@ func updateEpochMetrics(metrics module.ComplianceMetrics, snap protocol.Snapshot
 		return fmt.Errorf("could not get current epoch: %w", err)
 	}
 	// update epoch counter
-	counter, err := currentEpoch.Counter()
-	if err != nil {
-		return fmt.Errorf("could not get current epoch counter: %w", err)
-	}
-	metrics.CurrentEpochCounter(counter)
+	metrics.CurrentEpochCounter(currentEpoch.Counter())
 
-	currentEpochFinalView, err := currentEpoch.FinalView()
-	if err != nil {
-		return fmt.Errorf("could not update current epoch final view: %w", err)
-	}
-	metrics.CurrentEpochFinalView(currentEpochFinalView)
-
-	dkgPhase1FinalView, dkgPhase2FinalView, dkgPhase3FinalView, err := protocol.DKGPhaseViews(currentEpoch)
-	if err != nil {
-		return fmt.Errorf("could not get dkg phase final view: %w", err)
-	}
+	metrics.CurrentEpochFinalView(currentEpoch.FinalView())
 
-	metrics.CurrentDKGPhaseViews(dkgPhase1FinalView, dkgPhase2FinalView, dkgPhase3FinalView)
+	metrics.CurrentDKGPhaseViews(currentEpoch.DKGPhase1FinalView(), currentEpoch.DKGPhase2FinalView(), currentEpoch.DKGPhase3FinalView())
```

Comment:
I would just remove all the separating empty lines in between (and also the [comment](https://github.com/onflow/flow-go/blob/cfd9f9d2f2a7b5017e54800f8fd45e0c4030916a/state/protocol/badger/state.go#L851), since its trivial and we don't have comment for the other 2 `metrics` calls)